.gitignore

```diff
@@ -70,3 +70,4 @@ pythonenv*
 /clang/utils/analyzer/projects/*/RefScanBuildResults
 # automodapi puts generated documentation files here.
 /lldb/docs/python_api/
+/Debug/
```

Comment:
This seems accidental? I removed it in 994bdce654b980b4347c04d7b502a6348edd57a3

(Personally I place build directories in /tmp to prevent mistakes..)